### PR TITLE
feat: DIDexchange demo with public DIDs

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- Part of #795

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
the docs should be updated: https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/agent_cli.md

**Comment by sudeshrshetty** [comment]
> the docs should be updated: https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/agent_cli.md

done

File: cmd/aries-agentd/startcmd/start.go [hunk 1]
```diff
@@ -159,17 +176,20 @@ func Cmd(server server) (*cobra.Command, error) {
...
 	}
 
-	createFlags(startCmd, &webhookURLsFromCmdLine)
+	createFlags(startCmd, &webhookURLsFromCmdLine, &httpResolversFromCmdLine)
 
 	return startCmd, nil
 }
 
-func createFlags(startCmd *cobra.Command, webhookURLs *[]string) {
+// TODO need to get rid of extra args webhookURLs & httpResolvers
```
Comment (rolsonquadras) [inline cmd/aries-agentd/startcmd/start.go]: Issue # missing.

File: cmd/aries-agentd/startcmd/start_test.go [hunk 2]
```diff
@@ -380,20 +380,38 @@ func TestStartMultipleAgentsWithSameDBPath(t *testing.T) {
...
 
+func TestStartAriesErrorWithResolvers(t *testing.T) {
+	path, cleanup := generateTempDir(t)
+	defer cleanup()
+
+	testHostURL := randomURL()
+	testInboundHostURL := randomURL()
+
+	parameters := &agentParameters{&HTTPServer{}, testHostURL, testInboundHostURL,
+		"", path, "x", []string{}, []string{"http://sample.com"}}
```
Comment (troyronda) [inline cmd/aries-agentd/startcmd/start_test.go]: what does "x" mean?

File: cmd/aries-agentd/startcmd/start.go (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/aries-agentd/startcmd/start.go]: Created https://github.com/hyperledger/aries-framework-go/issues/799

File: cmd/aries-agentd/startcmd/start_test.go (hunk 2 above)
Comment (sudeshrshetty) [reply cmd/aries-agentd/startcmd/start_test.go]: just a non-empty label/agent name

File: cmd/aries-agentd/startcmd/start.go (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/aries-agentd/startcmd/start.go]: updated TODO

**Pull request merged**